apps/ledger-live-desktop/cryptoassets.md

```diff
@@ -19,6 +19,7 @@
 | Elrond | EGLD | YES | elrond |
 | Ethereum | ETH | YES | ethereum |
 | Ethereum Classic | ETC | YES | ethereum_classic |
+| Filecoin | FIL | YES | filecoin |
```

Comment:
Maybe we need to do something similar for LLM ? I tried to find the same file in ledger-live-mobile without success. Feel free to resolve if there is no equivalent on LLM

Reply:
I don't exactly know what this file is used for, but no equivalent on LLM indeed.